waiter/src/waiter/request_log.clj

```diff
@@ -46,8 +46,10 @@
 (defn response->context
   "Convert a response into a context suitable for logging."
   [{:keys [authorization/principal backend-response-latency-ns descriptor latest-service-id get-instance-latency-ns
-           handle-request-latency-ns instance status] :as response}]
-  (let [{:keys [service-id service-description]} descriptor]
+           handle-request-latency-ns headers instance status] :as response}]
+  (let [{:keys [service-id service-description]} descriptor
+        response-server (or (get headers "server")
+                            (get headers "Server"))]
```

Comment:
Aren't we always supposed to get lowercase headers?

Reply:
Changed to all lower-case "server" headers in our code.